src/HeuristicCompletion-Benchmarks/CoStaticBenchmarks.class.st

```diff
@@ -1,83 +1,21 @@
 "
-`CoStaticBenchmarks`, is designed to run completion benchmarks over a given **scope** (typically a class or package). It explores **completion suggestions** by trying out various prefix sizes (from 2 to 8 characters) and recording:
+`CoStaticBenchmarks` provides a benchmarking framework for evaluating the performance and accuracy of various sorting algorithms in the context of heuristic-based completion systems. It defines methods for executing benchmarks, collecting metrics such as completion accuracy, memory usage, and execution time, and generating statistical reports. The benchmarks can be executed on different sorter classes, and results are stored for further inspection and analysis.
+
+`CoStaticBenchmarks`, is designed to run completion benchmarks over a given **scope** (typically a class or package).
 
 1. **Accuracy** (whether the correct selector is found among the top 10 suggestions and at which index).
 2. **Timing** (the total and average time taken to compute suggestions for each prefix size).
+3. **Memory Usage** (the total and average memory consumed during benchmark execution).
+4. **Mean Reciprocal Rank (MRR)** (measures how high the correct suggestion appears in the ranked list).
+5. **Normalized Discounted Cumulative Gain (NDCG)** (evaluates ranking quality based on relevance distribution).
+6. **Rank Distribution** (counts how frequently the correct completion appears in each ranking position).
+7. **Recall@K** (measures how often the correct result appears within the top-K results).
 
-Below is a short overview of the main responsibilities and methods:
-
-## Main Responsibilities
-
-1. **Setup and Running (`run`, `runFor:`)**
```

Comment:
We should keep these comments because they explain the API. 
May be you move it in subclass but really we should not lose them.